app/services/youtube_service.rb

```diff
@@ -0,0 +1,26 @@
+class YoutubeService
+  def initialize(episode)
+    @episode = episode
+    @video = Yt::Video.new id: episode.video
+    Yt.configuration.api_key = ENV['TEST_CHANNELKEY']
```

Comment:
configuring in service isn't ok. I think you can make it with initializer

Reply:
Done